inst/rmarkdown/templates/arxiv/resources/template.tex

```diff
@@ -89,7 +94,7 @@
 
 $body$
 
-\bibliographystyle{unsrt}
+\bibliographystyle{$if(bibstyle)$$bibstyle$$else$unsrt$endif$}
```

Comment:
I would prefer to use `biblio-style` here to use the same variable name than default pandoc template for latex

* https://pandoc.org/MANUAL.html#biblatex-bibliographies
* https://github.com/jgm/pandoc-templates/blob/ea3841573fe83d8e101d70cbd78a1d6aa4bd6a21/default.latex#L363

```suggestion
\bibliographystyle{$if(biblio-style)$$biblio-style$$else$unsrt$endif$}
```

Is there a specific reason to use `bibstyle` ? 

I know `springer_article` use it but I think it should use also `biblio-style`.